pkg/cmd/experimental/policy/policy.go

```diff
@@ -79,23 +79,23 @@ func getUniqueName(basename string, existingNames *util.StringSet) string {
 	return string(util.NewUUID())
 }
 
-func getExistingRoleBindingForRole(roleNamespace, role, bindingNamespace string, client *client.Client) (*authorizationapi.RoleBinding, *util.StringSet, error) {
+func getExistingRoleBindingsForRole(roleNamespace, role, bindingNamespace string, client *client.Client) ([]*authorizationapi.RoleBinding, *util.StringSet, error) {
```

Comment:
Just feels like it leads to gorpy things like clients having to uniquify a name, possibly remove multiple identical bindings, etc.